### PR TITLE
Update android-deployment-scenarios-app-protection-work-profiles.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed contradicting statement as intune support two scenarios for BYOD

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/memdocs/issues/2436

File: memdocs/intune/apps/android-deployment-scenarios-app-protection-work-profiles.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ This feature applies to:
 
 ### Android Enterprise personally-owned work profiles
 
-Android Enterprise personally-owned work profiles are the core Android Enterprise deployment scenario and the only scenario targeted at BYOD use cases. The Android Enterprise personally-owned work profile is a separate partition created at the Android OS level that can be managed by Intune.
+Android Enterprise personally-owned work profiles is the core Android Enterprise deployment scenario . The Android Enterprise personally-owned work profile is a separate partition created at the Android OS level that can be managed by Intune.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/android-deployment-scenarios-app-protection-work-profiles.md:83]:
```suggestion
Android Enterprise personally-owned work profiles are the core Android Enterprise deployment scenario. The Android Enterprise personally-owned work profile is a separate partition created at the Android OS level that can be managed by Intune.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger17** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please merge this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**